src/test/java/teammates/test/cases/automated/FeedbackSessionClosedReminderTest.java

```diff
@@ -1,185 +0,0 @@
-package teammates.test.cases.automated;
```

Comment:
To be very frank, I have finished #6243 and even the issue afterwards which is to restore and revolutionize the automated actions tests that we have right now :p